httpcore5/src/main/java/org/apache/hc/core5/http/impl/bootstrap/AsyncRequester.java

```diff
@@ -27,12 +27,6 @@
 
 package org.apache.hc.core5.http.impl.bootstrap;
 
-import java.io.IOException;
```

Comment:
@clevertension Can we avoid moving imports around? It is not a big problem but it makes the change-set unnecessarily noisy.

Reply:
ok, i will adjust it